source/_integrations/opentherm_gw.markdown

```diff
@@ -60,12 +60,19 @@ The precision and floor_temperature settings that were supported in configuratio
 The OpenTherm Gateway can be further configured through the integration settings in the web interface
 The following options are available:
 {% configuration %}
-Precision:
-  description: "The desired precision for this device. Can be used to match your actual thermostat's precision. Set to `0` to use the default value for your unit preference."
+Read Precision:
+  description: "The desired read precision for this device. Used to display the current temperature on the climate entity. Can be used to match your actual thermostat's precision. Set to `0` to use the default value for your unit preference."
+  required: false
```

Comment:
Does `required` make sense for a dropdown?

Reply:
No, it does not, but there it no real option here.

This `configuration` block, should be converted into a `configuration_basic` block, which does not have all these additional (and required) fields.

Reply:
Will do. So I can lose the 'type' and 'required' fields. Should I keep the 'default' value? Or should it be just the description?

Reply:
Just the description, the default value is not part of the base (and not needed with UI configurations).